example-context/example-service/example-it/src/test/java/uk/gov/justice/services/example/cakeshop/it/CakeShopIT.java

```diff
@@ -1,59 +1,58 @@
 package uk.gov.justice.services.example.cakeshop.it;
 
-import static com.jayway.awaitility.Awaitility.await;
-import static com.jayway.jsonassert.JsonAssert.emptyCollection;
-import static com.jayway.jsonassert.JsonAssert.with;
-import static java.util.Arrays.asList;
-import static javax.json.Json.createArrayBuilder;
-import static javax.json.Json.createObjectBuilder;
-import static javax.ws.rs.client.Entity.entity;
-import static org.hamcrest.CoreMatchers.equalTo;
-import static org.hamcrest.CoreMatchers.hasItem;
-import static org.hamcrest.CoreMatchers.is;
-import static org.junit.Assert.assertThat;
-import static org.junit.Assert.fail;
-
+import liquibase.Liquibase;
+import liquibase.database.jvm.JdbcConnection;
+import liquibase.resource.ClassLoaderResourceAccessor;
+import org.apache.activemq.artemis.jms.client.ActiveMQConnectionFactory;
+import org.apache.commons.dbcp2.BasicDataSource;
+import org.apache.http.NameValuePair;
+import org.apache.http.client.utils.URIBuilder;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+import org.apache.http.impl.conn.PoolingHttpClientConnectionManager;
+import org.apache.http.message.BasicNameValuePair;
+import org.jboss.resteasy.client.jaxrs.ResteasyClientBuilder;
+import org.jboss.resteasy.client.jaxrs.engines.ApacheHttpClient4Engine;
+import org.junit.*;
+import org.junit.runners.MethodSorters;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import uk.gov.justice.services.eventsourcing.repository.jdbc.eventlog.EventLog;
 import uk.gov.justice.services.example.cakeshop.it.util.ApiResponse;
 import uk.gov.justice.services.example.cakeshop.it.util.StandaloneEventLogJdbcRepository;
 import uk.gov.justice.services.example.cakeshop.it.util.TestProperties;
 
+import javax.jms.*;
```

Comment:
wildcard imports and most changes are refactoring due to having a different code style. Suggest you reapply the code style now that you have the correct one.